tests/unit/test_pool.py

```diff
@@ -241,25 +241,36 @@ def test_spans_bind_get(self):
 
         with trace_call("pool.Get", SESSIONS[0]) as span:
             pool.get()
-            wantEventNames = [
-                "Acquiring session",
-                "Waiting for a session to become available",
-                "Acquired session",
-            ]
-            self.assertSpanEvents("pool.Get", wantEventNames, span)
 
-        # Check for the overall spans too.
+        span_list = self.get_finished_spans()
+        got_span_names = [span.name for span in span_list]
+        want_span_names = ["CloudSpanner.FixedPool.BatchCreateSessions", "pool.Get"]
+        print("got", got_span_names)
+        assert got_span_names == want_span_names
+
+        attrs = TestFixedSizePool.BASE_ATTRIBUTES.copy()
+        attrs["db.instance"] = ""
+
+        # Check for the overall spans.
+        self.assertSpanAttributes(
+            "CloudSpanner.FixedPool.BatchCreateSessions",
+            status=StatusCode.OK,
+            attributes=attrs,
+            span=span_list[0],
+        )
+
         self.assertSpanAttributes(
             "pool.Get",
+            status=StatusCode.OK,
             attributes=TestFixedSizePool.BASE_ATTRIBUTES,
+            span=span_list[-1],
         )
-
         wantEventNames = [
             "Acquiring session",
             "Waiting for a session to become available",
             "Acquired session",
         ]
-        self.assertSpanEvents("pool.Get", wantEventNames)
+        self.assertSpanEvents("pool.Get", wantEventNames, span_list[-1])
```

Comment:
any reason for doing span_list[-1] instead of span_list[1]?

Reply:
Yes:
a) pool.bind(database) came first before pool.get() hence produced the span "CloudSpanner.FixedPool.BatchCreateSessions"
b) "pool.Get" was the last span hence the -1 index